src/devices/video/snes_ppu.cpp

```diff
@@ -83,9 +83,9 @@
 #define SNES_CLIP_ALWAYS   3
 
 #define SNES_VRAM_SIZE        0x10000   /* 64kb of video ram */
-#define SNES_CGRAM_SIZE       0x202     /* 256 16-bit colours + 1 tacked on 16-bit colour for fixed colour */
+#define SNES_CGRAM_SIZE       0x200     /* 256 16-bit colours */
 #define SNES_OAM_SIZE         0x440     /* 1088 bytes of Object Attribute Memory */
-#define FIXED_COLOUR          256       /* Position in cgram for fixed colour */
+#define FIXED_COLOUR          0x900     /* Position in cgram for fixed colour */
```

Comment:
Is the comment still correct?  0x900 is too high to be an offset in a 256-word RAM.

Reply:
In palette entry, 256-word CGRAM area is 0x00-0xff, Direct colour layout is 0x100-8ff (256 per group).
Fixed colour offset is the topmost entry of palette now.